padertorch/configurable.py

```diff
@@ -21,12 +21,18 @@
 import importlib
 import inspect
 from pathlib import Path
+from functools import partial
```

Comment:
How about using `import functools` to avoid confusion with the keyword `partial`?